include/struct_pack/struct_pack/struct_pack_impl.hpp

```diff
@@ -263,7 +263,7 @@ constexpr auto get_types() {
     return declval<
         std::tuple<typename T::first_type, typename T::second_type>>();
   }
```

Comment:
`std::is_class_v<T>` is duplicate.

What happen if the members of structure is protect level after the commit accepted ?